index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Embedded HTML and CSS Project</title>
+        <link rel="stylesheet" href="styles.css">
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed&family=Roboto:wght@300;400&display=swap" rel="stylesheet">   
+    </head>
+
+    <body>
+
+        <div class="main-cont">
+
+            <nav class="nav-cont">
+
+                <div class="logo-cont">
+                    <div class="options-icon">
+                        <img src="images/options-icon.png">
+                    </div>
+                    <div>
+                        <img src="images/new-logo.jpg" class="yt-logo">
+                    </div>
+                </div>
+
+                <div class="search-cont">
+                    <form class="search-btn">
+                        <input type="text" placeholder="Search..">
+                        <button type="submit"><i class="fa fa-search"></i></button>
+                    </form>
+                </div>
+    
+                <div class="icons-cont">
+                    <div class="video-icon">
+                        <img src="images/video-icon.png">
+                    </div>
+                    <div class="apps-icon">
+                        <img src="images/apps-icon.png">
+                    </div>
+                    <div class= "bell-icon">
+                        <img src="images/bell-icon.png">
+                    </div>
+                    <div class="user-icon">
+                        <img src="images/user-icon.png">
+                    </div>
+                </div>
+     
+            </nav>
+    
+<!--------------------------- MAIN VIEW: Video - Description - videos/right-------->
```

Comment:
The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.